packages/embeds/embed-react/playwright/config/playwright.config.ts

```diff
@@ -1,4 +1,4 @@
-import { PlaywrightTestConfig, devices } from "@playwright/test";
+import type { PlaywrightTestConfig } from "@playwright/test";
```

Comment:
Opened this file and showed an error for which just saving it converted this automatically.